extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.js

```diff
@@ -62,6 +68,18 @@ export class AmpStorySubscriptions extends AMP.BaseElement {
       this.subscriptionService_ = subscriptionService;
       this.localizationService_ = localizationService;
 
+      const pages = this.win.document.querySelectorAll('amp-story-page');
+      const subscriptionsPageIndex = this.element.getAttribute(
+        'subscriptions-page-index'
+      );
+      this.storeService_.dispatch(
+        Action.SET_SUBSCRIPTIONS_PAGE_INDEX,
+        subscriptionsPageIndex > 2 &&
+          subscriptionsPageIndex !== pages.length - 1
+          ? subscriptionsPageIndex
+          : DEFAULT_SUBSCRIPTIONS_PAGE_INDEX
```

Comment:
Is the intended logic that the number must be between `DEFAULT_SUBSCRIPTIONS_PAGE_INDEX` and `pages.length`? So no one can set the subscription index to be 1 for example?

If that's the case, pls use clamp from `import {clamp} from '#core/math';` which makes it very easy to follow.